velox/common/caching/AsyncDataCache.cpp

```diff
@@ -676,6 +691,43 @@ bool AsyncDataCache::makeSpace(
   return false;
 }
 
+uint64_t AsyncDataCache::shrink(uint64_t targetBytes) {
+  VELOX_CHECK_GT(targetBytes, 0);
+
+  const uint64_t minBytesToEvict = 8UL << 20;
+  uint64_t evictedBytes{0};
+  for (int attempt = 0; attempt < 1; ++attempt) {
```

Comment:
I'm confused by this loop? Doesn't it run exactly once? And do we ever have attempt > 0 on L701?

Reply:
Oh, it is bug. Thanks and it is supposed to run twice. Thanks!